_eo_001_enemies/meracydian_clone.md

```diff
@@ -4,6 +4,8 @@ nickname: Meracydian Clone
 start_floor: 1
 end_floor: 9
 agro: 'Sight'
+hp: 415716
+attack_damage: 26000
```

Comment:
Ouch!

Reply:
In my second run I found one of these on floor 1 and decided to give it a try. Needless to say, that run didn't last long...